### PR TITLE
Filter out <none> images

### LINKED ISSUE #2626: Can't scan or delete anonymous images
When you select a `<none>:<none>` image you get a popup with "Error trying to scan <none>:latest".

Clicking "Ok" on the modal dialog then wipes out the "FATAL image scan error: ..." from the app window, so I can't copy & paste it.

You also can't delete them.

### PR DESCRIPTION
**Pull request by torchiaf**
Fixes #2626

Signed-off-by: Francesco Torchia <francesco.torchia@suse.com>

### CODE REVIEW

File: src/components/Images.vue [hunk 1]
```diff
@@ -333,6 +336,9 @@ export default {
     imageTag(tag) {
       return tag === '<none>' ? 'latest' : `${ tag.trim() }`;
     },
+    isNotNoneImage(row) {
+      return row.imageName && row.imageName !== '' && row.imageName !== '<none>';
```
Comment (rak-phillip) [inline src/components/Images.vue:340]: I was contemplating if testing only for `imageName` would be enough but it sounds like this is the preferred approach after reading through the issue.

Comment (rak-phillip) [inline src/components/Images.vue:340]: We should be able to safely make use of optional chaining and simplify the expression a little bit 🙂

```suggestion
      return row?.imageName !== '' && row?.imageName !== '<none>';
```

**Review by rak-phillip — COMMENTED** [review]
LGTM, just one minor suggestion 🙂

**Review by ericpromislow — DISMISSED** [review]
Change looks good, but the e2e tests failed at the start of four different test runs. Usually that's due to some startup flake and not the PR itself.

File: src/components/Images.vue (hunk 1 above)
Comment (torchiaf) [reply src/components/Images.vue]: Thanks Phil! Just pushed.

File: src/components/Images.vue [hunk 2]
```diff
@@ -333,6 +336,9 @@ export default {
     imageTag(tag) {
       return tag === '<none>' ? 'latest' : `${ tag.trim() }`;
     },
+    isNotNoneImage(row) {
+      return row?.imageName !== '' && row?.imageName !== '<none>';
```
Comment (ericpromislow) [inline src/components/Images.vue:340]: Even more concise:
```
return row.imageName && row.imageName !== '<none>';
```

**Review by ericpromislow — CHANGES_REQUESTED** [review]
Chance for more concise code possible

File: src/components/Images.vue (hunk 2 above)
Comment (torchiaf) [reply src/components/Images.vue]: > Chance for more concise code possible

Thanks Eric, it is fixed.

**Review by ericpromislow — APPROVED** [review]
LGTM

**Pull request merged**